tern/utils/constants.py

```diff
@@ -47,5 +47,7 @@
 mergedir = 'mergedir'
 # report file
 report_file = 'report.txt'
+# html report file
+html_file = 'report.html'
```

Comment:
This made me realize that the default output should be to console, but it looks like we do need a default file for certain output files. In this case, I wonder if this will work if we revert https://github.com/tern-tools/tern/pull/732?

Reply:
No, it will not work if we revert #732. The way that Tern was implementing printing to an output file was either print to file specified by user OR print to console. We would need a new fix.

Reply:
Some discussion in https://github.com/tern-tools/tern/issues/731 and proposal to merge this PR as-is since it adheres to current documentation around reporting.

Reply:
Sounds good to me